app/src/hooks/useSafeCookie.ts

```diff
@@ -14,7 +14,20 @@ export function useSafeCookie(
   }, [key]);
 
   const updateCookie = (newValue: string) => {
-    Cookies.set(key, newValue);
+    if (newValue === "") {
+      // Remove the cookie if empty value is provided
+      Cookies.remove(key, { path: '/' });
+      setValue(undefined);
+      return;
+    }
+
+    // Set cookie with proper options for persistence and security
+    Cookies.set(key, newValue, {
+      expires: 30, // 30 days
```

Comment:
nit: could set this using env vars; so that they can be modified/updated without redeployment (in the case we want to force-logout).

```suggestion

    // Set cookie with proper options for persistence and security
    const expiryDays = parseInt(process.env.NEXT_PUBLIC_COOKIE_EXPIRY_DAYS || '30', 10);
    Cookies.set(key, newValue, {
      expiryDays,
```